garak/buffs/paraphrase.py

```diff
@@ -68,11 +68,18 @@ def _get_response(self, input_text):
     def transform(
         self, attempt: garak.attempt.Attempt
     ) -> Iterable[garak.attempt.Attempt]:
-        yield self._derive_new_attempt(attempt)
-        paraphrases = self._get_response(attempt.prompt)
+        yield self._derive_new_attempt(
+            attempt
+        )  # why does this yield a copy of the original with no modification?
```

Comment:
paraphrase's goal is to add more variants of a prompt. e.g. if we dont have enough variations on a theme. here, we want more, so we retain the original. this should be a configurable behaviour, i don't remember if that feature was implemented yet though. its inclusion needn't block the pr